gnss_localizer/launch/nmea2tfpose.launch

```diff
@@ -5,6 +5,7 @@
 
   <node pkg="gnss_localizer" type="nmea2tfpose" name="nmea2tfpose" output="log">
     <param name="plane" value="$(arg plane)"/>
+    <param name="plane" value="0" if="$(arg sim_mode)"/>
```

Comment:
Where is the `sim_mode` argument coming from? It isn't defined above in this launch file.

Reply:
sim_mode comes from nmea2tfpose.yaml.
https://github.com/Autoware-AI/utilities/blob/master/autoware_launcher/plugins/refs/nmea2tfpose.yaml#L5

After running the following,  under computing tab, app next to nmea2tfpose has Simulation Mode, which is sim_mode.
$ roslaunch runtime_manager runtime_manager.launch

Reply:
I added arg sim_mode with default value "false". Running both the following cases works. 
1.roslaunch gnss_localizer nmea2tfpose.launch plane:=0
2.Running nmea2tfpose after activating simulation mode in Runtime Manager.